contrib/denom.json

```diff
@@ -0,0 +1,30 @@
+
+
+[{
+  "base": "uatom",
+  "denom_units": [
+    {
+      "aliases": [
+        "microatom"
+      ],
+      "denom": "uatom",
+      "exponent": 0
+    },
+    {
+      "aliases": [
+        "milliatom"
+      ],
+      "denom": "matom",
+      "exponent": 3
+    },
+    {
+      "aliases": [],
+      "denom": "atom",
+      "exponent": 6
+    }
+  ],
+  "description": "The native staking token of the Cosmos Hub.",
+  "display": "atom",
+  "name": "asdf",
```

Comment:
this is dummy info because you can't start a chain with an invalid genesis file and the problem that was originally addressed is that we can't export a valid genesis file until we fill these values. So for the upgrade test they have to be filled with something even though we're replacing the content with what _shoud_ have been there all along.